paasta_tools/cli/cmds/remote_run.py

```diff
@@ -76,34 +80,32 @@ def resourceOffers(self, driver, offers):
             self.status = 0
             driver.stop()
         else:
-            # driver.launchTasks(offer.id, tasks)
-            paasta_print("THIS IS NOT A DRILL!"
-                         "Would have launched: {task}".format(task=task))
-            self.status = 0
-            driver.stop()
+            driver.launchTasks(offer.id, [task])
 
 
     def statusUpdate(self, driver, update):
-        paasta_print("Mesos Scheduler: task %s is in state %d", update.task_id.value, update.state)
+        paasta_print("Mesos Scheduler: task %s is in state %d" % (update.task_id.value, update.state))
```

Comment:
there is a bunch more interesting info that comes from these messages. checkout https://github.com/Yelp/chronos/blob/master/src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosJobFramework.scala#L205 for the stuff I get for chronos